src/utils/tests.ts

```diff
@@ -0,0 +1,25 @@
+import fs from "fs";
+import MockAdapter from "axios-mock-adapter/types";
+
+export function logUnknownRequest(mock: MockAdapter) {
```

Comment:
@SferaDev Still very rought, but I am using your `mock.onAny()` here to log what I'd expect to find.